src/app/interfaces/ResponseObject.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2018 Amsterdam University of Applied Sciences (AUAS)
+ *
+ * This software is distributed under the terms of the
+ * GNU General Public Licence version 3 (GPL) version 3,
+ * copied verbatim in the file "LICENSE"
+ */
+
+export interface ResponseObject<T> {
+    apiVersion: string;
+    meta?: Meta;
+    data: {
+        [key: string]: any;
+        item: T;
+    };
+}
+
+export interface ResponseObjectCollection<T> {
```

Comment:
Maybe rename this interface to something more appropriate, since one might think this is a collection of ResponseObjects, while it is actually one response object with a collection of items.